README.md

```diff
@@ -54,7 +54,16 @@ The easiest way to run Arktos is to bring up a single-node cluster in your local
 
 ```
 cd $GOPATH/src/github.com/arktos
-hack/arktos-up.sh
+./hack/arktos-up.sh
+```
+
+The above command shows how to set up arktos with default network solution, bridge. With release 0.9, an advanced network solution,
+[Mizar](https://github.com/CentaurusInfra/mizar), is introduced into arktos. The integration with Mizar allows tenant pods/services to
```

Comment:
when using arktos to refer the project, it should be Arktos.

Reply:
Updated